### DISCUSSION TITLE
geonames cities is not a real city, more like a district or  town

### DISCUSSION THREAD

**Discussion by he2121** [immich-app/immich#13847] [General]
These cities are too small for me to recognize their locations at a glance. I am usually only familiar with places at the city level. However, the current display and search are at the smallest geoname level.

Is it more reasonable, like country-state-city-town(or anything).

**Comment by Moosbueffel** [comment]
For me it is wll, as it is. A country-state-city-town would be too roughly designed. e.g. I did a lot of travels in the same area. With your suggestion I can't  differentiate this.

But I found sometimes, that the city is wrong. I would wish to change the city-name. This will than also solve your problem :). Perhaps a feature request.
